go/vt/schemadiff/table.go

```diff
@@ -454,6 +454,15 @@ func NewCreateTableEntity(env *Environment, c *sqlparser.CreateTable) (*CreateTa
 	return entity, nil
 }
 
+func (c *CreateTableEntity) ColumnDefinitionEntities() []*ColumnDefinitionEntity {
```

Comment:
I don't see where this is used ...

Reply:
You're right. This is preparation work for a later PR (WIP: https://github.com/vitessio/vitess/pull/16256).